libvast/src/system/passive_partition.cpp

```diff
@@ -161,12 +161,16 @@ indexer_actor passive_partition_state::indexer_at(size_t position) const {
   // Deserialize the value index and spawn a passive_indexer lazily when it is
   // requested for the first time.
   const auto* qualified_index = flatbuffer->indexes()->Get(position);
+  if (!qualified_index || qualified_index->index())
+    return {};
   if (auto value_index
       = unpack_value_index(*qualified_index->index(), *container)) {
     indexer = self->spawn(passive_indexer, id, std::move(value_index));
     return indexer;
   }
-  VAST_ERROR("{} failed to deserialize value index at {}", *self, position);
+  VAST_WARN("passive-partition ({}) failed to deserialize value index for "
+            "field {}",
+            id, qualified_index->field_name()->string_view());
```

Comment:
`unpack_value_index` itself prints errors in case the binary or legacy deserializer fails, does it really make sense to print a second warning in that case?

Reply:
I would rather consider that warning to be misplaced, as it's impossible to investigate if you don't know which partition the failure originated from. Should we remove that one?

Reply:
Done. Mind approving the PR if that is fine with you?